src/index.html

```diff
@@ -15,8 +19,100 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="/"
+      >
+        <img
+          class="logo-img"
+          src="images/logo.png"
+          alt="moyo-logo"
```

Comment:
The `alt` attribute for the logo image should be more descriptive to improve accessibility. Consider using a more specific description of the image content instead of 'moyo-logo'.